src/main/java/org/usfirst/frc4904/robot/commands/IndexOne.java

```diff
@@ -0,0 +1,28 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.subsystems.Indexer;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+import org.usfirst.frc4904.robot.subsystems.Shooter;
+import org.usfirst.frc4904.robot.commands.OpenIndexer;
+import org.usfirst.frc4904.robot.commands.CloseIndexer;
+
+public class IndexOne extends SequentialCommandGroup {
+  /**
+   * Open the flippers, run the run up belt until the limit switch is activated,
+   * then close the flippers.
+   * 
+   * @param indexer
+   * @param shooter
+   */
+  public IndexOne(Indexer indexer, Shooter shooter) {
+    super(new OpenIndexer(indexer), new MotorConstant("RunIndexer", indexer.liftBelts, Indexer.DEFAULT_LIFT_SPEED),
+        new WaitUntilCommand(() -> {
+          return shooter.limitSwitch.get();
+        }), new CloseIndexer(indexer));
```

Comment:
should the indexer stay open until the shooter's limit switch returns true? i'm not sure exactly where that is on the robot, but i'd imagine we would want the indexer to close before that if possible.

Reply:
we need to clarify with design about the positioning of the limit switch